client/gutenberg/extensions/simple-payments/edit.js

```diff
@@ -462,7 +466,7 @@ const mapSelectToProps = withSelect( ( select, props ) => {
 		: undefined;
 
 	return {
-		hasPublishAction: get( getCurrentPost(), [ '_links', 'wp:action-publish' ], false ),
+		hasPublishAction: !! get( getCurrentPost(), [ '_links', 'wp:action-publish' ], false ),
```

Comment:
`false` default value is redundant ( `!! undefined === false` ) but that's fine.